### ISSUE TITLE
Mystery Text Discussion of fc.txt

### ISSUE THREAD

**Issue by ebeshero** [newtfire/introDH-Hub#51]
Post your screenshots and discuss your findings about fc.txt here!

**Comment by TheMightyDaltho** [comment]
From my ngrams and words used in abundance in this text, It's pretty clear to me this is the novel Frankenstein by Mary Shelly. This assumption is based on two things I noticed. First when I was looking at length 6 n-grams, I noticed the phrase "with you on yourr wedding night" which is infamous threat made by the monster to Dr. Frankenstein when he refused to make him a wife. To double check I checked to see how many times monster was used and it turned out to be 31 times. Based on that information I'm fairly certain this is Frankenstein.

**Comment by sydneymcdonough** [comment]
After using AntConc it surprised me that the most commonly used word besides the most common words such as 'the' and 'to' was the word 'Elizabeth'. This reading was multiple articles "ALPHONSE FRANKENSTEIN" sent to different people in his life. The most common phrase was 'with you on your wedding night'. Putting the most frequent word and phrase and previous knowledge of reading the story of Frankenstein we know the letter about the Dr making Frankenstein a wife named Elizabeth is the most pertinent article in the reading.  

# 
<img width="1254" alt="Screen Shot 2022-03-22 at 1 02 36 AM" src="https://user-images.githubusercontent.com/98784979/159411841-3300967b-bd77-492b-bb24-a1561e38f3a0.png">
<img width="1254" alt="Screen Shot 2022-03-22 at 1 02 11 AM" src="https://user-images.githubusercontent.com/98784979/159411843-f009e7ad-21d7-43cf-a2b4-b47634e9c64b.png">
<img width="1254" alt="Screen Shot 2022-03-22 at 1 01 38 AM" src="https://user-images.githubusercontent.com/98784979/159411844-25c3e2cf-1ccd-4ff0-8362-5a83b15e3865.png">
<img width="1254" alt="Screen Shot 2022-03-22 at 1 01 30 AM" src="https://user-images.githubusercontent.com/98784979/159411845-e5c3a5d1-0eaf-4db7-ba2e-3fe471c9787a.png">
<img width="1254" alt="Screen Shot 2022-03-21 at 11 14 20 PM" src="https://user-images.githubusercontent.com/98784979/159411846-9db187cf-0b75-4cb7-b670-dcf10649630a.png">
<img width="1251" alt="Screen Shot 2022-03-21 at 9 48 11 PM" src="https://user-images.githubusercontent.com/98784979/159411847-e2b82e31-df26-4530-b624-a2c47a8b4da8.png">
<img width="1280" alt="Screen Shot 2022-03-17 at 12 35 11 PM" src="https://user-images.githubusercontent.com/98784979/159411848-f1af5ddf-e9aa-42ad-9166-126dec5abc59.png">
#

**Comment by ebeshero** [comment]
@sydneymcdonough Where is Elizabeth showing up in Antconc? I don’t see it in your screen captures?

None of the names you mention are in your screen shots, so I am not understanding this analysis.

**Comment by mmw6204** [comment]
So initially reading some of this text left me a little confused. I use AntConc and was quite surprised with the results. "Be with you on your wedding" and "with you on your wedding night" were the most used phrases in the text. When I searched up Frankenstein to see what would pop up, I became slightly more confused. Each phrase was used in a different tense which meant that most of them were quotes and people were saying Frankenstein's name in the letter. When I used Voyant to analyze the text, it showed man being the most used word (131 times). This was quite interesting to click through and see all the different ways the word man was used throughout the text. 
![fcsnip](https://user-images.githubusercontent.com/98792692/159423065-8a030aa8-b477-4db4-923c-52d9990ceb7b.PNG)
![fcsnip2](https://user-images.githubusercontent.com/98792692/159423116-4e8e0f35-a74f-4009-adb8-ba856c8afbd9.PNG)
![fcsnip3](https://user-images.githubusercontent.com/98792692/159423119-0c81d910-2640-4840-8d9d-e348e7ffa072.PNG)
